bcrs-ui/package.json

```diff
@@ -11,36 +11,36 @@
   "dependencies": {
     "@mdi/font": "^4.5.95",
     "axios": "^0.18.0",
-    "core-js": "^3.4.3",
+    "core-js": "^3.4.4",
     "css-loader": "^0.28.11",
     "keycloak-js": "^6.0.0",
```

Comment:
Don't need 'keycloak.js' (as it's now imported by sbc-common-components).

Reply:
Thank you, will remove!